_posts/2020-02-10-calculate-tech-debt-using-skunk-on-github-actions.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title: "How to Calculate Tech Debt Using Skunk on GitHub Actions"
+date: 2020-02-10 16:16:00
+categories: ["tech-debt"]
```

Comment:
@etagwerker let's add `code-quality` too. Your other articles about Skunk are under that category